src/base/tests/test_util.py

```diff
@@ -24,6 +26,19 @@
 from odoo.addons.base.maintenance.migrations.util.exceptions import MigrationError
 
 
+@contextmanager
+def without_testing():
+    thread = threading.current_thread()
+    testing = getattr(thread, "testing", False)
```

Comment:
Always `False` with new implementation.
```suggestion
    testing = getattr(modules.module, "current_test", False) or getattr(thread, "testing", False)
```